### PR TITLE
MGMT-16587: Bump runc version to 1.1.12 to Mitigate CVE-2024-21626

### PR DESCRIPTION
**Pull request by paul-maidment**
We are bumping runc to 1.1.12 to mitigate the CVE

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [x] Automation (CI, tools, etc)
- [x] Cloud
- [x] Operator Managed Deployments
- [x] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: deploy/s3/deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
         app: object-store
     spec:
       containers:
-        - image: quay.io/cloudservices/minio:latest
```
Comment (adriengentil) [inline deploy/s3/deployment.yaml:19]: should not be part of this PR?

Comment (paul-maidment) [reply deploy/s3/deployment.yaml]: Good point, I have removed this.

**Review by adriengentil — APPROVED** [review]

**Pull request merged**